randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,24 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          # You always need morph
+          Morph Ball
+          Any of the following:
+              # Either flash or grapple with a trivial wall jump, or space jump directly
+              Flash Shift or Grapple Beam or Space Jump
+              All of the following:
+                  # Speed boost into spider magnet surface or beginner walljump
+                  Speed Booster
+                  Spider Magnet or Walljump (Beginner)
+              All of the following:
+                  # Midair sideways cross bomb into spider magnet surface,  beginner damage boost or onto frozen enemy
+                  Lay Cross Comb
+                  Any of the following:
+                      Spider Magnet or Damage Boost (Beginner)
+                      Stand on Frozen Enemy (Intermediate) and Shoot Ice Missile
```

Comment:
So you can get a wall jump off with just cross? Probably should be wall jump beginner then if the timing is any bit tight. The damage boost would also need paired with the Stand on Frozen Enemies

So,
Spider Magnet or Wall Jump (Beginner)
Damage Boost (Beginner) and Stand on Frozen Enemies (Intermediate) and Shoot Ice Missile

Reply:
I'll try to figure out if the simple cross bomb trick can be done with a wall jump. But the damage boosts and standing on frozen I'm not super interested in checking personally, but I'll add it in as per requested.

Reply:
No need to check for damage boost. the damage is done together with the frozen enemy, so they must be combined, not separated.